services/cis/infrastructure/cdk/src/constructs/contract-dynamo-construct.ts

```diff
@@ -41,7 +41,7 @@ export class ContractDynamoConstruct extends cdk.Construct {
         });
     }
 
-    addGSI() {
+    private addGSI() {
```

Comment:
do we want to pull index names into cdk.json context? I guess it might be a later things too when table structures are actually more clear.

Reply:
Good point. attributes names & Gsi name will be needed in developer code as well, unless we pass them as environment variables to Lambda or put it in ssm(and pass the name), they will need to hardcode them in their code. should we ask developers or Larry ?

Reply:
sure

Reply:
@bradshawlb

Reply:
@bradshawlb  confirmed that we don't need to pass dynamo table name or GSI names as environment variables to Lambda.